frontend/viewer/src/lib/about/AboutDialog.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import { mdiInformationVariantCircle } from '@mdi/js';
+  import { Button, Dialog, Toggle } from 'svelte-ux';
+  import type { Readable } from 'svelte/store';
+  import Markdown from 'svelte-exmarkdown';
+  import NewTabLinkRenderer from './NewTabLinkRenderer.svelte';
+  import { onMount } from 'svelte';
+
+  export let text: Readable<string>;
+
+  let toggle: Toggle;
+
+  onMount(() => {
+    if (!localStorage.getItem('suppressAbout')) {
```

Comment:
this will work for the viewer, however it doesn't work for FW Lite as local storage does not persist for the moment

Reply:
This one is the "comment", right?

That's a good point 😕.
Can we tackle that problem when it becomes a problem?

Reply:
yeah it's just a comment. Not really sure what to do about it, maybe we can shim our own local storage. Or maybe when we migrate to the blazor hybrid web view then local storage will start working? But it'll still not work for linux users as we're not going that path there